processor/main.py

```diff
@@ -36,14 +37,110 @@ def write_to_vdb(mapped: list) -> None:
             if w is not None:
                 w.close()
 
+def write_to_neo4j(events) -> None:
```

Comment:
I think this function here would be better off in a class elsewhere where it can be reused or made into a library etc.

It's also doing multiple things, ie. It connects to the database, it constructs the data object, and then it writes to the db. Try to encapsulate each of those separate actions in their own logical constructs so they can be composed here.

One mental mnemonic I use is "Do I feel compelled to write a comment describing what the next lines are doing?" If yes, it's a separate method the name of which should describe what I would have put in that comment. 

I think it's counterintuitive and maybe even contrary to some of what gets taught, but the best code has only docstrings, never online comments.

Reply:
@knordstrom I agree with this!
Where do you think I could create a Class for this? I also wanted to create a Class for each node type like Person and Event. What do you think about this?